master/about.md

```diff
@@ -64,6 +64,8 @@ Heptio Ark treats object storage as the source of truth. It continuously checks
 
 This allows restore functionality to work in a cluster migration scenario, where the original backup objects do not exist in the new cluster.
 
+Likewise, if a backup object exists in Kubernetes but not in the object storage, it will be deleted from Kubernetes since this means it is no longer in use.
```

Comment:
I would probably word the first part `Likewise, if a backup object exists in Kubernetes but not in object storage, ...`.

Reply:
Ok cool. I like the attention to detail!